electron_app/electron.ts

```diff
@@ -1025,11 +1025,16 @@ async function startRuntime(): Promise<void> {
     const sqlitePath = getProcessEngineDatabaseFolder();
     const logFilepath = getProcessEngineLogFolder();
 
-    runtimeProcess = fork(
+    const pathToRuntime = path.join(
+      __dirname,
+      '..',
+      '..',
+      '..',
       './node_modules/@process-engine/process_engine_runtime/bin/index.js',
```

Comment:
Der Pfad sollte auch in Segmenten an `path.join` übergeben werden, damit der Pfad der am Ende rauskommt auf jeder Plattform konsistent bleibt.

```suggestion
      'node_modules',
      '@process-engine',
      'process_engine_runtime',
      'bin',
      'index.js',
```